### PR TITLE
fix(keyExpectedValue): cloudformation-aws queries convert to a recommendation rather than a current status - stage 2

### PR DESCRIPTION
**Pull request by liorj-orca**
…endation rather than a current status - stage 2

Closes #

**Proposed Changes**
-
-
-

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/cloudFormation/aws/directory_service_microsoft_ad_password_set_to_plaintext_or_default_ref/query.rego [hunk 1]
```diff
@@ -21,7 +21,7 @@ CxPolicy[result] {
 		"resourceName": "n/a",
 		"searchKey": sprintf("Parameters.%s.Default", [paramName]),
 		"issueType": "IncorrectValue",
-		"keyExpectedValue": sprintf("Parameters.%s.Default isn't defined", [paramName]),
+		"keyExpectedValue": sprintf("Parameters.%s.Default should not defined", [paramName]),
```
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/directory_service_microsoft_ad_password_set_to_plaintext_or_default_ref/query.rego:24]:
```suggestion
"keyExpectedValue": sprintf("Parameters.%s.Default should not be defined", [paramName]),
```

File: assets/queries/cloudFormation/aws/ec2_permissive_network_acl_protocols/query.rego [hunk 2]
```diff
@@ -14,7 +14,7 @@ CxPolicy[result] {
 		"resourceName": cf_lib.get_resource_name(resource, name),
 		"searchKey": sprintf("Resources.%s.Properties.Protocol", [name]),
 		"issueType": "IncorrectValue",
-		"keyExpectedValue": sprintf("'Resources.%s.Properties.Protocol' should be hould be either 6 (for TCP), 17 (for UDP), 1 (for ICMP), or 58 (for ICMPv6, which must include an IPv6 CIDR block, ICMP type, and code)", [name]),
+		"keyExpectedValue": sprintf("'Resources.%s.Properties.Protocol' should be either 6 (for TCP), 17 (for UDP), 1 (for ICMP), or 58 (for ICMPv6, which must include an IPv6 CIDR block, ICMP type, and code)", [name]),
```
Comment (rafaela-soares) [inline assets/queries/cloudFormation/aws/ec2_permissive_network_acl_protocols/query.rego:17]: I notice a typo in keyActualValue. Can you fix it, please? Thank you so much, in advance.

"protocol different **then** 6" to "protocol different **than** 6"

**Review by rafaela-soares — CHANGES_REQUESTED** [review]
Hello again, @liorj-orca!

Thank you so much for another PR 🚀

This PR is also failing in the E2E tests. If you need any help, let me know 😊

**Comment by gabriel-cx** [comment]
Hello, @liorj-orca!
Feel free to ping me if you need something from our side in order to close this PR! ⛵
If you prefer, we can close it in our side.

**Review by cxMiguelSilva — APPROVED** [review]
LGTM

**Review by cxMiguelSilva — APPROVED** [review]

**Review by rafaela-soares — APPROVED** [review]

**Pull request merged**